samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/evaluation/QuestionScoreListener.java

```diff
@@ -720,7 +720,7 @@ else if ("false".equals(answerText)) {
 					else {
 						if (gdata.getRationale() != null
 								&& !gdata.getRationale().trim().equals(""))
-							rationale = "\nRationale: " + gdata.getRationale();
+							rationale = "\n" + ContextUtil.getLocalizedString(MSG_BUNDLE, "rationale") + " " + gdata.getRationale();
```

Comment:
The resource loader is already initialized in this class with that bundle it's better to just use it vs calling ContextUtil

```suggestion
							rationale = "\n" + rb.getString("rationale") + " " + gdata.getRationale();
```

Reply:
I would update the other uses of ContextUtil there as it is reloading bundle data every time it is called vs initializing once.

Reply:
@ern Just updated